OpenEdXMobile/src/main/java/org/edx/mobile/base/BaseSingleFragmentActivity.java

```diff
@@ -44,6 +44,15 @@ protected void onCreate(Bundle savedInstanceState) {
         super.setToolbarAsActionBar();
     }
 
+    /**
+     * It will add the custom toolbar in the activity's layout.
+     * <p>
+     * Toolbar addition will be done by finding a {@link FrameLayout} with id
+     * {@link R.id#toolbar_container R.id.toolbar_container} and then inflating a custom toolbar layout
+     * in it. Custom toolbar layout will be get from {@link BaseSingleFragmentActivity#getToolbarLayoutId()}
```

Comment:
```java
     /**
     * It will add the custom toolbar in the activity's layout.
     * <p>
     * Toolbar addition will be done by finding a {@link FrameLayout} with id
     * {@link R.id#toolbar_container R.id.toolbar_container} and then inflating a custom toolbar layout
     * in it. Custom toolbar layout will be obtained from {@link BaseSingleFragmentActivity#getToolbarLayoutId()}
     * function.
     * </p>
     */
```